### PR TITLE
Add support for esp32-s3-eye devkit

### PR DESCRIPTION
**Pull request by jepler**
This depends on the PIDs that have been requested from espressif: 
 * https://github.com/espressif/usb-pids/pull/54

### CODE REVIEW

File: ports/espressif/build/CMakeFiles/cmake.check_cache [hunk 1]
```diff
@@ -0,0 +1 @@
+# This file is generated by cmake for dependency checking of the CMakeCache.txt file
```
Comment (hathach) [inline ports/espressif/build/CMakeFiles/cmake.check_cache:1]: this file should not be added.

**Review by hathach — COMMENTED** [review]
look great, you will need to add it into the ci build as well https://github.com/adafruit/tinyuf2/blob/master/.github/workflows/build_esp32.yml

File: ports/espressif/build/CMakeFiles/cmake.check_cache (hunk 1 above)
Comment (jepler) [reply ports/espressif/build/CMakeFiles/cmake.check_cache]: I force-pushed to remove this file.

**Comment by jepler** [comment]
I confirmed that the built firmware works for loading circuitpython uf2 files

**Comment by jepler** [comment]
In theory support for the LCD could be added but it's not a priority for me

**Comment by hathach** [comment]
> In theory support for the LCD could be added but it's not a priority for me

not a problem at all, we could do it later when having time. Please add it into the ci build and this is good to merge. https://github.com/adafruit/tinyuf2/blob/master/.github/workflows/build_esp32.yml

File: .github/workflows/build_esp32.yml [hunk 2]
```diff
@@ -66,7 +66,9 @@ jobs:
         - 'adafruit_qtpy_esp32s3'
         - 'bpi_leaf_s3'
         - 'espressif_esp32s3_devkitc_1'
+        - 'espressif_esp32s3_devkitc_1'
```
Comment (hathach) [inline .github/workflows/build_esp32.yml:69]: This line is duplicated to above line of 68

Comment (jepler) [reply .github/workflows/build_esp32.yml]: oops, removed now!

**Review by hathach — APPROVED** [review]
perfect now

**Pull request merged**